docs/docs/get_started/quickstart.mdx

```diff
@@ -471,6 +472,7 @@ While the first part of this guide was intended to be run in a Jupyter Notebook,
 Install with:
 ```bash
 pip install "langserve[all]"
+pip install "fastapi"
```

Comment:
this is installed via the above installation

Reply:
got error package not found after installing **langserve[all]** when following the guides 
platform: Mac Pro - M2 chip
Python : 3.9
Pip: 23.3.2

Reply:
removed this since it has been added in langserve[all] dependency list